openhands/cli/commands.py

```diff
@@ -214,7 +214,7 @@ async def init_repository(current_dir: str) -> bool:
     if repo_file_path.exists():
         try:
             content = await asyncio.get_event_loop().run_in_executor(
-                None, read_file, repo_file_path
+                None, read_file, str(repo_file_path)
```

Comment:
I think this is perhaps too strongly typed. I the return value for `Path` is `str | None`, then `str(repo_file_path)` is always a string

mypy wouldn't complain because `None` would become `"None"` (in the event we didn't have the if statement before)

Not blocking but worth double checking imo